services/core-web/src/components/common/DocumentTable.js

```diff
@@ -47,11 +57,55 @@ const defaultProps = {
   additionalColumnProps: [],
   documentColumns: null,
   documentParent: null,
-  defaultSortKeys: ["upload_date", "dated"], // keys to sort by when page loads
+  isLoaded: false,
+  noSubTableExpandableRows: false,
+  defaultSortKeys: ["upload_date", "dated", "update_timestamp"], // keys to sort by when page loads
   view: "standard",
   canArchiveDocuments: false,
 };
 
+const renderFileLocation = (documentTypeCode) => {
+  let location = "N/A";
+  switch (documentTypeCode) {
+    case "PRM":
+      location = "Primary Document";
+      break;
+    case "SPT":
+      location = "Spatial Component";
+      break;
+    case "SPR":
+      location = "Supporting Document";
+      break;
+  }
```

Comment:
See src/constants/strings.js

We do have constants for MAJOR_MINES_APPLICATION_DOCUMENT_TYPE_CODE (this one might need pretty labels) and EMPTY_FIELD that we can refer to.

Reply:
I ended up adding a new variable to the strings.js

```
export const MAJOR_MINES_APPLICATION_DOCUMENT_TYPE_CODE_LOCATION = {
  PRM: "Primary Document",
  SPT: "Spatial Component",
  SPR: "Supporting Document",
};
```

I think MAJOR_MINES_APPLICATION_DOCUMENT_TYPE_CODE is being used elsewhere and changing the keys in this object would affect the places it is being used, but if I didn't change the keys I think I would need to adjust text case (upper to lower) and add text like Document or Component to what we want. I think just adding another variable probably just simplifies things.